fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java

```diff
@@ -881,8 +881,8 @@ private Function checkPolymorphicFunction(Function fn, Type[] paramTypes) {
         } else if (typeElement != null) {
             typeArray = new ArrayType(typeElement);
         } else {
-            LOG.warn("could not determine polymorphic type because has unknown types");
-            return null;
+            typeElement = Type.NULL;
```

Comment:
here must be null type?

Reply:
In fact, it is also possible to use Boolean instead of Null, and even other data types can be used